packages/dds/tree/src/codec/codec.ts

```diff
@@ -333,7 +333,23 @@ export function withSchemaValidation<
  * Versions of Fluid Framework client packages.
  * @remarks
  * Used to express compatibility requirements by indicating the oldest version with which compatibility must be maintained.
+ *
+ * When no compatibility impacting change landed in a given version, the value associated with its enum entry may point to the older version which its fully compatible with.
+ * Note that this can change if a future version of the framework introduces an option to use something which is only supported at a particular version, then the values of the enum may shift,
```

Comment:
Nit
```suggestion
 * Note that this can change if a future version of the framework introduces an option to use something which is only supported at a particular version. In which case, the values of the enum may shift,
```